cli/cmd/stat.go

```diff
@@ -53,16 +53,19 @@ func newCmdStat() *cobra.Command {
   Examples:
   * deploy
   * deploy/my-deploy
-  * deploy my-deploy
-  * ns/my-ns
+  * rc/my-replication-controller
+	* ns/my-ns
+	* authority
+	* au/my-authority
```

Comment:
You're using tab indents on lines 57-59, but they should be spaces for consistency with the rest of the help message.